cdc/owner/changefeed_test.go

```diff
@@ -138,6 +138,9 @@ func createChangefeed4Test(ctx cdcContext.Context, t *testing.T) (*changefeed, *
 	}, func() DDLSink {
 		return &mockDDLSink{}
 	})
+	cf.newScheduler = func(ctx cdcContext.Context, startTs uint64) (scheduler, error) {
+		return newSchedulerV1(), nil
+	}
```

Comment:
Is this test only available with old scheduler?

Reply:
The correctness of the scheduler is not the goal of these tests. The old scheduler is like a "mock scheduler" here.